### PR TITLE
#1 Submit Button Propagates List Item

### PR DESCRIPTION
**Pull request by rrgayhart**
Brenna and I are hot fire 🌶 

Completes phase 1 - bullet 1 referenced in issue #1 with a quick refactor

### CODE REVIEW

**Comment by rrgayhart** [comment]
@stevekinney - OMG WHAT EVEN IS CODE GOOD LORD OMG

File: script.js [hunk 1]
```diff
@@ -1 +1,17 @@
...
+var $itemInput = $('.grocery-list-form--item');
+var $quantityInput = $('.grocery-list-form--quantity');
+var $list = $('.grocery-list');
 
+$submitButton.on('click', function(e) {
+  e.preventDefault();
+  var item = $itemInput.val();
+  var quantity = $quantityInput.val();
+  var listItem = createListItem(item, quantity);
+
```
Comment (martensonbj) [inline script.js]: Do we really need this extra line? @rrgayhart :1st_place_medal:

Comment (rrgayhart) [reply script.js]: Aw dang.

**Comment by stevekinney** [comment]
@rrgayhart Just saw this. I assume you and Brenna don't need a code review anymore.

**Pull request merged**